lls/nd.c

```diff
@@ -21,6 +21,7 @@
 
 #include <rte_ip.h>
 
+#include "gatekeeper_net.h"
```

Comment:
Once the other comments are addressed, we'll no longer need to include this header.